File: 基礎題庫/Contents/b884/b884.py
import sys

for n in sys.stdin:
    n = int(n)

    for i in range(n):
        x, y = map(int, input().split())

        yee = 100 - x - y

        if yee > 0 and yee <= 30:
            print("sad!")
        elif yee > 30 and yee <= 60:
            print("hmm~~")
        elif yee > 60 and yee < 100:
            print("Happyyummy")
        else:
            print("evil!!")

File: 基礎題庫/Contents/b884/b884.cpp
#include<iostream>

using namespace std;

int main(void)
{
	int n, yee, x, y;
	
	while(cin >> n)
	{
		for(int i = 0; i < n; i++)
		{
			cin >> x >> y;
			
			yee = 100 - x - y;
			
			if(0 < yee && yee <= 30)
				cout << "sad!" << endl;
			else if(yee > 30 && yee <= 60)
				cout << "hmm~~" << endl;
			else if(yee > 60 && yee < 100)
				cout << "Happyyummy" << endl;
			else
				cout << "evil!!" << endl;
		}
	}
	
	return 0;
}
